pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -168,6 +168,10 @@ func (pl *TestPlugin) Less(*framework.QueuedPodInfo, *framework.QueuedPodInfo) b
 }
 
 func (pl *TestPlugin) Score(ctx context.Context, state *framework.CycleState, p *v1.Pod, nodeName string) (int64, *framework.Status) {
+	// NOTE: The tests expect some non-zero latency to occur while running these methods. On Windows, time.Now()
```

Comment:
Could this also result in product issues?

Reply:
Separated into a different PR: https://github.com/kubernetes/kubernetes/pull/113253